library/systemd/src/lib/yast2/service_widget.rb

```diff
@@ -199,6 +218,12 @@ def action_widget
       )
     end
 
+    def valid_action?(value)
```

Comment:
The logic of this method is kind of duplicated in `#action_items`. I would try to unify it.